code/game/machinery/vending.dm

```diff
@@ -1003,21 +1008,23 @@
 
 /obj/machinery/vending/security
 	name = "\improper SecTech"
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts."
+	product_ads = "Defend yourself!;Help others!;All you need for a secure facility right here!;Your weapons are right here.;Handcuffs!;Did you know that 'Freeze, scumbag!' is the most standardised quote broadcasted by this machine!;Try your best to apprehend all criminals and subject them to the law.;Reminder to always use non-lethals first!;Why not have a donut?"
 	icon_state = "sec"
 	icon_deny = "sec-deny"
 	icon_vend = "sec-vend"
 	vend_delay = 14
 	base_type = /obj/machinery/vending/security
-	req_access = list(ACCESS_SECURITY)
+	req_access = list(ACCESS_SECURITY_LVL1)
```

Comment:
please double check that this still allows all sec to use it, the way our access is handled is a bit shit and needs to be reworked so its possible this only allows for level one sec guards to access this.

Reply:
it allows.